srlife/damage.py

```diff
@@ -130,10 +290,23 @@ def calculate_element_log_reliability(
           volumes:        element volumes
           material:       material model  object with required data
         """
-        raise NotImplementedError("Method is pure virtual in base class")
+
+        # Equivalent stress raied to exponent mv
+        flat = (
+            self.calculate_flattened_eq_stress(
+                mandel_stress,
+                self.temperatures,
+                self.material,
+                self.A,
```

Comment:
`self.A`, `self.dalpha`, and `self.dbeta` shouldn't be part of the function call.